packages/graphiql/test/schema.js

```diff
@@ -19,7 +21,18 @@ const {
   GraphQLString,
   GraphQLID,
   GraphQLList,
-} = require('graphql');
+  GraphQLDeferDirective,
+  GraphQLStreamDirective,
+  specifiedDirectives,
+  version,
+} = graphql;
+
+const enableExperimentalIncrementalDelivery =
+  !version.startsWith('15') && !version.startsWith('16');
```

Comment:
```suggestion
const enableExperimentalIncrementalDelivery = parseInt(version) > 16
```
maybe?
<img width="157" alt="image" src="https://github.com/user-attachments/assets/a77539c3-ac53-44ba-8f6a-b5027f7f75ef">

Reply:
70bbc6cd6559c051f129a3b299806e01ede96c08